delivery_automatic_print/models/report_print.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def button_validate(self):
+        res = super(StockPicking, self).button_validate()
+        for pickings in self:
```

Comment:
I am unsure if this is missing of that or not: I see no check that these documents have been created before. Question is: Do you even want such a check?

Reply:
@nikunjantala We would be glad if you could provide feedback.

Reply:
@manengel Yes, We are checking that if that picking is delivery order, then system send Shipping label attached(UPS or Deutsche post) and Delivery order report to Printer for print.

Reply:
@manengel Yes, We are checking that if that picking is delivery order, then system send Shipping label attached(UPS or Deutsche post) and Delivery order report to Printer for print.